### PR TITLE
subject(SumArray):add readme

### PR DESCRIPTION
**Pull request by Hamzaelkhatri**

### CODE REVIEW

**Review by zainabdnaya — APPROVED** [review]
All seems to me good

**Comment by davidrobert99** [comment]
@Hamzaelkhatri in your last commit is missing the  "Here is a possible program to test your function:".

**Review by davidrobert99 — APPROVED** [review]

File: subjects/arraysum/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+```go
+func ArraySum(numbers []int) int {
+    // your code here
+}
+```
+
+### Usage
+
+```go
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply subjects/arraysum/README.md:20]: it's already added

File: subjects/arraysum/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+and the output should be:
+
+```console
+$ go run . 
+15
+0
+-15
+3
+```
```
[the remark this answers is not in the document]
Comment (Hamzaelkhatri) [reply subjects/arraysum/README.md]: I guess there is already a newline at the end of the file

**Pull request merged**